test/ILLink.RoslynAnalyzer.Tests/TestCaseUtils.cs

```diff
@@ -48,16 +49,25 @@ public static IEnumerable<object[]> GetTestData (string testSuiteName)
 							yield return new object[] { m, attrs };
 						}
 					}
+					if (node is AccessorDeclarationSyntax a) {
+						var attrs = a.AttributeLists.SelectMany (al => al.Attributes.Where (IsWellKnown)).ToList ();
+						if (attrs.Count > 0) {
+							yield return new object[] { a, attrs };
+						}
+					}
 				}
 
 				static bool IsWellKnown (AttributeSyntax attr)
 				{
 					switch (attr.Name.ToString ()) {
-					// Currently, the analyzer's test infra only understands these attributes when placed on methods.
+					// Currently, the analyzer's test infra only understands these attributes when placed on methods and properties.
 					case "ExpectedWarning":
 					case "LogContains":
 					case "LogDoesNotContain":
-						return attr.Ancestors ().OfType<MemberDeclarationSyntax> ().First ().IsKind (SyntaxKind.MethodDeclaration);
+						var ancestor = attr.Ancestors ().OfType<MemberDeclarationSyntax> ().First ();
+						if (ancestor.IsKind (SyntaxKind.MethodDeclaration) || ancestor.IsKind (SyntaxKind.PropertyDeclaration) || ancestor.IsKind (SyntaxKind.GetAccessorDeclaration) || ancestor.IsKind (SyntaxKind.SetAccessorDeclaration))
+							return true;
+						return false;
```

Comment:
Nit: I think it helps readability to have one line between the `if () <exp>; <exp>;`. I've noticed this pattern in the codebase, although I didn't find any coding style doc (searched in runtime repo) that clearly states this.